demo/README.md

```diff
@@ -0,0 +1,23 @@
+# VM Definitions for vagrant 
+
+## adding new box 
+
+to add a new demo add a new folder with following structure
+
+```
+vmdefs/enabled/centos6-drools-guest/
+├── ansible.cfg
+├── playbook.yml
+└── Vagrantfile
+```
+
+## enabling boxes 
+
+create symlink of a box from ./available to ./enabled e.g.
+to enable drools app one would do following, note the '../'
+```
+[vmindru@vmutil vmdefs]$ ln -s ../available/centos6-drools-guest/ ./enabled/
```

Comment:
not really when you do ln -s you need to specify the relative path, your example actually works cause both your current location and the target share same relative path to the source file. 

```
[vmindru@vmutil some_dir]$ pwd
/tmp/some_dir
[vmindru@vmutil some_dir]$ echo content > ../source-dir/file                                                                                                                                                        
[vmindru@vmutil some_dir]$ ln -s ../source-dir/file ../target-dir/target-dir2/file                                                                                                                                  
[vmindru@vmutil some_dir]$ cat ../target-dir/target-dir2/file 
cat: ../target-dir/target-dir2/file: No such file or directory
[vmindru@vmutil some_dir]$ ls ../target-dir/target-dir2/file 
../target-dir/target-dir2/file
[vmindru@vmutil some_dir]$ ln -s ../../source-dir/file ../target-dir/target-dir2/relative_link_file
[vmindru@vmutil some_dir]$ cat ../target-dir/target-dir2/relative_link_file
content
[vmindru@vmutil some_dir]$ 
```